samples/sdl_joystick_gamepad/Makefile

```diff
@@ -0,0 +1,10 @@
+DEBUG = y
+XBE_TITLE = sdl_joystick_gamepad
+GEN_XISO = $(XBE_TITLE).iso
+SRCS += $(CURDIR)/main.cpp
+NXDK_DIR = $(CURDIR)/../..
+NXDK_SDL = y
+
+all_local: all
```

Comment:
nit: This shouldn't be necessary for simple examples like this one? Do the other samples do it?